rfc/rfc013-vc-irma-proof.md

```diff
@@ -237,9 +252,11 @@ The contents of the schemes can be viewed on Github: [pbdf](https://github.com/p
 
 ### 4.4 Revocation
 
-Although IRMA supports revocation for credentials, this isn't usable for digital signatures.
-The credential will be revoked at a certain timestamp, any signatures made in the past are still valid. 
+Although IRMA supports revocation for credentials, the revocation check is only applied during issuance.
+An IRMA credential revoked today will not invalidate any signatures done in the past.
```

Comment:
"any signatures done in the past"

I think the correct wording is "generated"/"created" in the past.